src/server/bg_services/namespace_monitor.js

```diff
@@ -169,6 +195,7 @@ class NamespaceMonitor {
             const container_client = conn.getContainerClient(target_bucket);
             await container_client.deleteBlob(block_key);
         } catch (err) {
+            this._fix_error_object(err);
```

Comment:
Here we are inside `test_blob_resource(nsr)`, are you sure you are using the AWS SDK here?

Reply:
We can remove it.
just wanted to be 100%